src/core/currency/wallet/currency-wallet-api.ts

```diff
@@ -65,6 +69,8 @@ const fakeMetadata = {
   notes: ''
 }
 
+export const PARENT_TOKEN_ID = ''
+
```

Comment:
I wonder if this line should live in `currency-wallet-files.ts`, since it's related to the on-disk representation.